### ISSUE TITLE
[Effect Suggestion] EMP

### ISSUE THREAD

**Issue by throwaway1234567890123456789** [gta-chaos-mod/ChaosModV#1063]
**EMP**

**Duration if timed effect (long or short timed?)**
long 

**Effect Description**
anything that electronic in the area does not work at all for the duration. Planes, helocopters, phones, cars, trucks, HUD/Radar, etc.

**Comment by anto4440** [comment]
planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. Hud shouldnt be disabled, but phone should. Stun gun and railgun should be disabled. 
oh, and one more thing: make it instant so that all loaded objects become disabled, with phone and such for a period of time (30 secs maybe) interesting idea

**Comment by pipecek** [comment]
> 
> 
> planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. Hud shouldnt be disabled, but phone should. Stun gun and railgun should be disabled.
> oh, and one more thing: make it instant so that all loaded objects become disabled, with phone and such for a period of time (30 secs maybe) interesting idea

Gasoline cars DO require power...

**Comment by anto4440** [comment]
> > planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. Hud shouldnt be disabled, but phone should. Stun gun and railgun should be disabled.
> > oh, and one more thing: make it instant so that all loaded objects become disabled, with phone and such for a period of time (30 secs maybe) interesting idea
> 
> Gasoline cars DO require power... "Battery: The battery provides electricity to start the engine and power vehicle electronics/accessories. ... Fuel pump: A pump that transfers fuel from the tank to the engine's fuel injection system via the fuel line. Fuel tank (gasoline): This tank stores gasoline on board the vehicle until it's needed by the engine."

Old cars didnt need those?

**Comment by pipecek** [comment]
> 
> 
> > > planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. Hud shouldnt be disabled, but phone should. Stun gun and railgun should be disabled.
> > > oh, and one more thing: make it instant so that all loaded objects become disabled, with phone and such for a period of time (30 secs maybe) interesting idea
> > 
> > 
> > Gasoline cars DO require power... "Battery: The battery provides electricity to start the engine and power vehicle electronics/accessories. ... Fuel pump: A pump that transfers fuel from the tank to the engine's fuel injection system via the fuel line. Fuel tank (gasoline): This tank stores gasoline on board the vehicle until it's needed by the engine."
> 
> Old cars didnt need electricity?
> (that was actually what i meant, it was just terribly explained)

oh ok! But most GTA car are modern and do infact require electricity

**Comment by anto4440** [comment]
> > > > planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. Hud shouldnt be disabled, but phone should. Stun gun and railgun should be disabled.
> > > > oh, and one more thing: make it instant so that all loaded objects become disabled, with phone and such for a period of time (30 secs maybe) interesting idea
> > > 
> > > 
> > > Gasoline cars DO require power... "Battery: The battery provides electricity to start the engine and power vehicle electronics/accessories. ... Fuel pump: A pump that transfers fuel from the tank to the engine's fuel injection system via the fuel line. Fuel tank (gasoline): This tank stores gasoline on board the vehicle until it's needed by the engine."
> > 
> > 
> > Old cars didnt need electricity?
> > (that was actually what i meant, it was just terribly explained)
> 
> oh ok! But most GTA cars are modern and do infact require electricity

Yeah exactly, that why i clarified how dumb of me it was not to write that in the original comment.

**Comment by LordRygar** [comment]
> planes and helis would go down, but cars wouldnt. I mean... electric cars would, but gasoline cars dont require any power. 


Actually, modern cars would be disabled, since they have computers in them.  But Classic Cars wouldn't be effected.
